cmd/antrea-agent/agent.go

```diff
@@ -80,7 +82,7 @@ func run(o *Options) error {
 
 	ovsBridgeClient := ovsconfig.NewOVSBridge(o.config.OVSBridge, o.config.OVSDatapathType, ovsdbConnection)
 	ovsBridgeMgmtAddr := ofconfig.GetMgmtAddress(o.config.OVSRunDir, o.config.OVSBridge)
-	ofClient := openflow.NewClient(o.config.OVSBridge, ovsBridgeMgmtAddr)
+	ofClient := openflow.NewClient(o.config.OVSBridge, ovsBridgeMgmtAddr, o.config.FeatureGates[string(features.AntreaProxy)])
```

Comment:
You should use `features.DefaultFeatureGate.Enabled(features.AntreaProxy)`.
But actually you don't have to create another flag in every class for it and can just use the global DefaultFeatureGate in implementation. For example: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/pods/helpers.go#L50

Reply:
Thanks for your information, updated.

Reply:
forgot to update?

Reply:
Sorry, I'm sure I updated this time.